storage/innobase/handler/ha_innodb.cc

```diff
@@ -1319,6 +1319,17 @@ free_share(
 /*=======*/
 	INNOBASE_SHARE*	share);		/*!< in/own: share to free */
 
+
+/************************************************************************/ /**
+Calls free_share and assign NULL to share. */
```

Comment:
it is not necessary to use old decoration
you could write the new doxygen comment header in 5.7 too

```
/** Calls free_share and assign NULL to share
@param[in,out]    share  table share to free */
```